2560.py

```diff
@@ -0,0 +1,15 @@
+import pymc3 as pm
+with pm.Model() as model:
```

Comment:
this should be a function in `pymc3/tests/test_text_backend.py`.  Something like

```
def test_wont_dump_mismatched_vars():
    with pm.Model() as model:
        a = pm.Gamma('a', mu=10.0, sd=2.0)
        b = pm.Gamma('b', mu=10.0, sd=2.0)

        partial_trace = pm.sample(draws=100, tune=0, trace=[model.a, model.a_log__])
        assert len(partial_trace.varnames) == 2
    
        with pytest.raises(ValueError):
            pm.backends.text.dump('trace.text', partial_trace)
```